src/Card/CardExpandable.js

```diff
@@ -17,15 +17,27 @@ function getStyles() {
 
 class CardExpandable extends Component {
   static propTypes = {
+    closeIcon: PropTypes.node,
     expanded: PropTypes.bool,
     onExpanding: PropTypes.func.isRequired,
+    openIcon: PropTypes.node,
     style: PropTypes.object,
   };
 
   static contextTypes = {
     muiTheme: PropTypes.object.isRequired,
   };
 
+  getCorrectIcon(isExpanded, propIconOpen, propIconClose) {
+    if (isExpanded) {
+      if (propIconOpen) return propIconOpen;
+      return <OpenIcon />;
```

Comment:
I'm sorry I can't understand what do you mean with "using defaultProps". Do you mean the function default props so that if propIconOpen and close are undefined it will do that automatically? Something Like this: getCorrectIcon(isexpanded, propIconOpen=<OpenIcon/>, propIconClose=<CloseIcon/> ) 

Sorry for the question but I'm pretty new in the react world